frontend/src/api/annonars.ts

```diff
@@ -0,0 +1,19 @@
+const API_BASE_URL =
+  import.meta.env.MODE == 'development' ? '//localhost:8080/proxy/annonars' : '/proxy/annonars'
```

Comment:
Maybe we should an `export const API_BASE_PREFIX` in a more central module (`api/common.ts`?) and import this from there. then use `const API_BASE_URL = \`${API_BASE_PREFIX}/\`` here?